PencilCase/PencilCase/scripts/Navigator.ts

```diff
@@ -10,6 +10,7 @@ import { Orders } from './Pages/Orders';
 import { BatchOrderDetails } from './Pages/BatchOrderDetails';
```

Comment:
Just Ctrl + k + d this file, so you would find a lot of changes, but actually, just updated the method below with comments.